src/sgame/sg_bot_local.h

```diff
@@ -125,6 +120,7 @@ enum bot_skill
 
 	// fighting skills
 	BOT_B_BASIC_FIGHT, // doesn't do anything as of now
+	BOT_B_FAST_AIM, // doesn't do anything as of now
```

Comment:
"does not do anything as for now" this should block the merge of this PR.

Reply:
This was a copy-paste error. Good catch.